crt_portal/cts_forms/forms.py

```diff
@@ -2028,63 +2028,104 @@ class PrintActions(Form):
     )
 
 
-class BulkDispositionForm(Form, ActivityStreamUpdater):
-    EMPTY_CHOICE = 'Multiple'
-    assigned_section = ChoiceField(
-        label='Section',
-        widget=ComplaintSelect(attrs={
-            'class': 'usa-select crt-dropdown__data',
-            'disabled': 'disabled',
-        }),
-        choices=add_empty_choice(SECTION_CHOICES_WITHOUT_LABELS, default_string=EMPTY_CHOICE),
-        required=False
-    )
-    retention_schedule = MultipleChoiceField(
-        required=False,
-        label='Retention schedule',
-        choices=add_empty_choice(RETENTION_SCHEDULE_CHOICES, default_string=EMPTY_CHOICE),
-        widget=ComplaintSelect(attrs={
-            'class': 'usa-select crt-dropdown__data',
-            'disabled': 'disabled',
-        }),
-    )
+class BulkDispositionForm(ModelForm, ActivityStreamUpdater):
 
-    def get_initial_values(self, record_query, keys):
-        """
-        Given a record query and a list of keys, determine if a key has a
-        singular value within that query. Used to set initial fields
-        for bulk update forms.
-        """
-        # make sure the queryset does not order by anything, otherwise
-        # we will have difficulty getting distinct results.
-        query = record_query.order_by()
-        for key in keys:
-            values = query.values_list(key, flat=True).distinct()
-            if values.count() == 1:
-                yield key, values[0]
+    def field_changed(self, field):
+        # if both are Falsy, nothing actually changed (None ~= "")
+        old = self.initial.get(field, None)
+        new = self.cleaned_data.get(field, None)
+        if not old and not new:
+            return False
+        return old != new
 
-    def __init__(self, query, *args, user=None, **kwargs):
+    @cached_property
+    def changed_data(self):
+        return [
+            field_name
+            for field_name
+            in super().changed_data
+            if self.field_changed(field_name)
+        ]
+
+    class Meta:
+        model = ReportDispositionBatch
+        fields = ['disposed_by', 'disposed_count', 'create_date', 'proposed_disposal_date']
+
+    def setup_disposed_by(self):
+        disposed_by = self.user
+        value = f'{disposed_by.first_name} {disposed_by.last_name}' if disposed_by.first_name and disposed_by.last_name else ''
```

Comment:
if these get super long, wrapping them in parens helps the formatter:

```suggestion
        value = (
            f'{disposed_by.first_name} {disposed_by.last_name}'
            if disposed_by.first_name and disposed_by.last_name
            else ''
        )
```